src/main/java/kitchenpos/domain/Product.java

```diff
@@ -1,33 +1,59 @@
 package kitchenpos.domain;
 
 import java.math.BigDecimal;
-
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import kitchenpos.domain.vo.Price;
+
+@Entity
 public class Product {
+
+    private static final int NAME_LENGTH_MAXIMUM = 255;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @Column(nullable = false)
     private String name;
-    private BigDecimal price;
+    @Embedded
+    private Price price;
 
-    public Long getId() {
-        return id;
+    public Product(final Long id, final String name, final BigDecimal price) {
+        validateName(name);
+        this.id = id;
+        this.name = name;
+        this.price = Price.from(price);
     }
 
-    public void setId(final Long id) {
-        this.id = id;
+    protected Product() {
     }
 
-    public String getName() {
-        return name;
+    public Product(final String name, final BigDecimal price) {
+        this(null, name, price);
     }
```

Comment:
```suggestion
    public Product(final String name, final BigDecimal price) {
        validateName(name);
        this.name = name;
        this.price = Price.from(price);
    }
```

```java
public Product(final Long id, final String name, final BigDecimal price) {
        validateName(name);
        this.id = id;
        this.name = name;
        this.price = Price.from(price);
    }
```
해당 생성자는 테스트를 위한 생성자인가요?

Reply:
실수로 남겨놓아서, 지웠습니다 ㅎㅎ 감사해용